### ISSUE TITLE
[Bug] `table` class is with Tailwindcss

### ISSUE THREAD

**Issue by runyasak** [saadeghi/daisyui#1066]
# Overview

There have table component in Daisyui and using `table` as class name. By the way, `table` class is using in Tailwindcss too.
The result is the attributes from both classes are override.

# Screenshots
<img width="447" alt="image" src="https://user-images.githubusercontent.com/6861191/187085432-3204966d-db5e-4c2f-ab4c-69c54e7dfe59.png">

# Actual Result
`table` class from Daisyui is overlapping with Tailwind.

# Expected Result
Rename css class for table component that not duplicate with Tailwindcss.

**Comment by saadeghi** [comment]
I know that Tailwind CSS also has a `table` class name but I don't see any conflicts.  
- `.table` from Tailwind CSS applies `display: table`
- `.table` from daisyUI applies design decisions to a `<table>` element that already should be `display: table`

By default, a `<table>` already has `display: table` style from the user agent so I don't see how that could cause problems.
>![Screen Shot 2022-08-29 at 2 42 32 AM](https://user-images.githubusercontent.com/7342023/187099710-62223168-1c37-4577-8db2-ad7bc0b14e44.png)

Let me know if you have any questions.

**Comment by runyasak** [comment]
Hi, @saadeghi. Thank you for your reply and information. 😁

I know as your information that `<table>` has `display: table` as attribute.

Actually, I have found this case in my own project.
This project was using Tailwindcss, I installed Daisyui when I take charge this project.

There have some component that is not table but using `display: table`, that is modal.

I was searching and found this example, that would be the reference for the modal in project:
https://codesandbox.io/s/github/vuejs/v2.vuejs.org/tree/master/src/v2/examples/vue-20-modal-component?from-embed

From this tutorial:
https://v2.vuejs.org/v2/examples/modal.html

However, using `display: table` for creating modal is not the good practice.

I give you my apologize that I declare this issue as `bug`.
I just want to discuss and share some idea that there have some cases like this or not.

For now, I have resolved by override css manually.

By the way, I really love this project and help me to deliver many projects so far.

Thank you for discussion. 😁
